app/templates/events/event_list.html

```diff
@@ -69,22 +69,22 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
     </thead>
     <tbody>
         {% for event in events %}
-          <tr>
```

Comment:
Why did you remove this? Also, if it was intended to be deleted and it will not be put back make user you also remove its closing tag on line 106